tests/apps/demultiplex/test_read_sample_sheet.py

```diff
@@ -183,14 +183,15 @@ def test_get_sample_internal_ids_from_sample_sheet(
     flow_cell_type: Type[FlowCellSample] = FlowCellSampleBCLConvert,
 ):
     """Test that getting sample internal ids from a sample sheet returns a unique list of strings."""
-    # GIVEN a path to a sample sheet with only valid samples
-
-    # WHEN getting the valid sample internal ids
-    sample_internal_ids: List[str] = get_sample_internal_ids_from_sample_sheet(
-        sample_sheet_path=novaseq6000_bcl_convert_sample_sheet_path,
+    # GIVEN a sample sheet with only valid samples
+    sample_sheet: SampleSheet = get_sample_sheet_from_file(
+        infile=novaseq6000_bcl_convert_sample_sheet_path,
```

Comment:
```suggestion
        sample_sheet_path=novaseq6000_bcl_convert_sample_sheet_path,
```
out of scope likely

Reply:
I'll do it together with the other refactoring in a separate PR 👍